NativeImport/ExampleNativeImportFeed.xml

```diff
@@ -79,6 +63,143 @@
                 <Featured>false</Featured>
                 <DisplayLocale>en_US</DisplayLocale>
             </Review>
+
+            <!-- sample review featuring attributes -->
+
+            <Review id="11" removed="false">
+                <ModerationStatus>APPROVED</ModerationStatus>
+                <UserProfileReference id="miketester112">
+                    <ExternalId>miketester112</ExternalId>
+                    <DisplayName>supermike</DisplayName>
+                    <Anonymous>false</Anonymous>
+                    <HyperlinkingEnabled>false</HyperlinkingEnabled>
+                </UserProfileReference>
+                <Title>Best product ever!</Title>
+                <ReviewText>This product truly changed my life; I don't know what I'd do without it.</ReviewText>
+                <Rating>5</Rating>
+                <IpAddress>64.94.228.137</IpAddress>
+                <Recommended>true</Recommended>
+                <UserEmailAddress>supermike@example.com</UserEmailAddress>
+                <ReviewerLocation>Austin, TX</ReviewerLocation>
+                <SubmissionTime>2012-01-23T12:59:25.000-06:00</SubmissionTime>
+                <Featured>false</Featured>
+                <DisplayLocale>en_US</DisplayLocale>
+
+                <!-- review feedbacks: -->
```

Comment:
Can you add a blurb above each of these sections that explain what they are. The inline comments for feedback are good, but what is a Context Data Value? What's a Rating Value?

Unfortunately, our feed doesn't use the same jargon as our other tools, so a quick description in-feed would be beneficial.